### PR TITLE
Add a "how to"-guide for template engine

### PR DESCRIPTION
**Pull request by simon-johansson**

### CODE REVIEW

File: docs/how-to/template-engine.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    const html = ejs.render(inlineHTML, { name: "Simon" });
+    resp.setHeader("Content-Type", "text/html");
+    resp.end(html);
+  },
+);
+```
+
+## Static files
+
+In the above example we are serving a stylesheet from the `/public` path. We use the `api.static` function to serve all files in the `./assets` directory under the `/public` path prefix.
```
Comment (marcuskohlberg) [inline docs/how-to/template-engine.md:108]: Above example? Or do you mean below?

Comment (simon-johansson) [reply docs/how-to/template-engine.md]: I updated this sentence to make it more clear

**Review by marcuskohlberg — DISMISSED** [review]

File: docs/how-to/template-engine.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+const inlineHTML = `
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="/public/styles.css" >
+  </head>
+  <body>
+    <h1>Static Inlnie HTML Example</h1>
```
Comment (eandre) [inline docs/how-to/template-engine.md:90]:
```suggestion
    <h1>Static Inline HTML Example</h1>
```

**Comment by simon-johansson** [comment]
@eandre done!

**Review by eandre — APPROVED** [review]

**Pull request merged**